package.json

```diff
@@ -19,7 +19,10 @@
   "module": "dist/module/index.js",
   "types": "dist/module/index.d.ts",
   "sideEffects": false,
-  "repository": "supabase/supabase-js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/supabase/supabase-js.git"
```

Comment:
I'm not sure if anything depended on `repository` previously. If it did we may need to check if the new format doesn't break those.

Reply:
Yah, good point. semantic-release should be able to understand the old format. so I'll revert to that. Thanks.